vocabularies/PDF.xml

```diff
@@ -0,0 +1,96 @@
+      </Term>
+
+      <ComplexType Name="FeaturesType">
+        <Annotation Term="Common.Experimental" />
+        <Property Name="DocumentDescriptionReference" Type="Edm.String" Nullable="false">
+          <Annotation Term="Core.IsURL" Bool="true" />
+          <Annotation Term="Core.Description" String="Reference of the Service for the DocumentDescription" />
+        </Property>
+        <Property Name="DocumentDescriptionCollection" Type="Edm.String" Nullable="false">
+          <Annotation Term="Core.Description" String="Collection of DocumentDescription" />
+        </Property>
+        <Property Name="ArchiveFormat" Type="Edm.Boolean" DefaultValue="false">
+          <Annotation Term="Core.Description" String="PDFA conformant format" />
+        </Property>
+        <Property Name="Signature" Type="Edm.Boolean" DefaultValue="false">
+          <Annotation Term="Core.Description" String="Signing the document" />
+        </Property>
+        <Property Name="CoverPage" Type="Edm.Boolean" DefaultValue="false">
+          <Annotation Term="Core.Description" String="Cover Page" />
+        </Property>
+        <Property Name="FontName" Type="Edm.String">
+          <Annotation Term="Core.Description" String="Font name" />
+        </Property>
+        <Property Name="FontSize" Type="Edm.Int16">
+          <Annotation Term="Core.Description" String="Font size" />
+        </Property>
+        <Property Name="Margin" Type="Edm.Int16">
+          <Annotation Term="Core.Description" String="Margin size" />
+        </Property>
+        <Property Name="Border" Type="Edm.Int16">
+          <Annotation Term="Core.Description" String="Border size of the table" />
+        </Property>
+        <Property Name="FitToPage" Type="Edm.Boolean" DefaultValue="false">
+          <Annotation Term="Core.Description" String="Fit to page" />
+          <Annotation Term="Core.LongDescription">
+            <String>If this option is selected, the font size is automatically selected in such a way that all columns of a table fit on one page. Other layout options like margin, border and composite cell spacing are adapted accordingly, with respect to the chose scaling factor.</String>
+          </Annotation>
+        </Property>
+        <Property Name="ResultSizeDefault" Type="Edm.Int32">
+          <Annotation Term="Core.Description" String="Default result size" />
+          <Annotation Term="Core.LongDescription">
+            <String> Default result size for PDF documents. Used if a the $top has not been provided.</String>
+          </Annotation>
+        </Property>
+        <Property Name="ResultSizeMaximum" Type="Edm.Int32">
+          <Annotation Term="Core.Description" String="Maximum result size" />
+          <Annotation Term="Core.LongDescription">
+            <String>Max result size for PDF documents. Used if $top has been provided and $top > ResultSizeMaximum</String>
+          </Annotation>
+        </Property>
```

Comment:
The client is forced to perform the paging by sending hundreds(?) of requests with suitable `$top` values and organize the hundreds(?) of downloaded PDF documents. This is unusable for customers who want to use PDF/A as an archive format.

Reply:
This depends if the service likes to protect the server from a DoD attack. It is not a must have to switch on DefaultPaging (enabled in RAP).


This was discussed with S/4 and the default paging size for PDF's are higher than for normal data requests.